app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -37,6 +37,20 @@ function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyServ
 
       if (currentSelectedKinds && (Object.keys(currentSelectedKinds).length != Object.keys($scope.kinds).length)) {
         $scope.kinds = currentSelectedKinds;
+      } else {
+        remove_hierarchy = ['Container', 'ContainerGroup', 'ContainerReplicator', 'ContainerService',
```

Comment:
(because missing var, improbable formatting, and easier to see if its one entry per line, sorted alphabetically)